substrafl/evaluation_strategy.py

```diff
@@ -18,10 +18,10 @@ def __init__(
 
         Args:
             test_data_nodes (List[TestDataNode]): nodes on which the model is to be tested.
-            eval_frequency (Optional[int]): The model will be tested every ``eval_frequency`` rounds. Default to 1.
-                Set to None to activate eval_rounds only.
+            eval_frequency (Optional[int]): The model will be tested every ``eval_frequency`` rounds.
+                Set to None to activate eval_rounds only. Default to None.
             eval_rounds (Optional[List[int]]): If specified, the model will be tested on the index of a round given
-                in the rounds list. Set to None to activate eval_frequency only.
+                in the rounds list. Set to None to activate eval_frequency only. Default to None.
```

Comment:
Shouldn't you raise an error if both values are left to None? And specify that in the doc?

Reply:
It was already checked and present il the docstring. Cf `At least one of eval_frequency or eval_rounds must be defined.` and `ValueError: both eval_rounds and eval_frequency cannot be None at the same time`